parsers/nmparser/parse_lst_file.go

```diff
@@ -54,6 +55,54 @@ func parseOFV(line string, ofvDetails OfvDetails) OfvDetails {
 	return ofvDetails
 }
 
+func setLargeConditionNumber(lines []string, start int, largeConditionNumber *bool) {
+
+	// go until line of ints
+	for i, line := range lines[start:] {
+		sub := strings.Trim(line, " ")
+		if len(sub) > 0 {
+			vals := strings.Fields(sub)
+			if len(vals) > 1 {
+				one, err := strconv.Atoi(vals[0])
+				if err == nil && one == 1 {
+					two, err := strconv.Atoi(vals[1])
+					if err == nil && two == 2 {
+						start = start + i
+						break
+					}
+				}
+			}
+		}
+	}
+
+	// go until blank line
+	for i, line := range lines[start:] {
+		sub := strings.Trim(line, " ")
+		if len(sub) == 0 {
+			start = start + i + 1
+			break
+		}
+	}
+
+	var eigenvalues []float64
+	for _, s := range strings.Fields(lines[start]) {
+		eigenvalue, err := strconv.ParseFloat(s, 64)
+		if err == nil {
+			eigenvalues = append(eigenvalues, eigenvalue)
+		}
+	}
+
+	if len(eigenvalues) >= 2 {
+		sort.Float64s(eigenvalues)
+		ratio := eigenvalues[len(eigenvalues)-1] / eigenvalues[0]
+		// TODO: get largeConditionNumber threshold from config
+		// or derive. something like (number of parameters) * 10
+		if ratio > 1000.0 {
```

Comment:
we should probably expose this better, so it can be controlled via configuration, so for now would elevate it to a param